content/sensu-enterprise/2.8/guides/snmp-sensu-guide.md

```diff
@@ -0,0 +1,252 @@
+
+_WARNING: Before proceeding any further, you’ll need to restart your `sensu-client` process_
+
+{{< highlight shell >}}
+sudo systemctl restart sensu-client{{< /highlight >}}
+
+You can confirm that the client is now listening on the correct port by performing the following command:
+
+{{< highlight shell >}}
+sudo netstat -plunt | grep 1062{{< /highlight >}}
+
+At this point, it’s worth noting that the configuration provided here is a basic, bare-minimum configuration. There are additional options you can add to your SNMP extension configuration to suit your needs. We cover those [here](#additional-snmp-extension-options). But to get a general sense of how SNMP traps function with Sensu, continue reading below.
+
+# Testing the SNMP Extension
+- So far, we’ve done the following:
+- Installed the SNMP trap extension
+- Configured the extension
+- Configured the SNMP trap receiver
+- Confirmed that the SNMP trap receiver is listening on our host
+
+Now comes the fun part: testing the extension to make sure that our trap receiver is working as expected. If you’ve met the prerequisites, you should already have the `net-snmp-utils` package installed on your host, which has the `snmptrap` command that we’ll be using in our tests. 
+
+Let’s start by generating a test trap:
+
+{{< highlight shell >}}
+sudo snmptrap -v 2c -c sensutest localhost:1062 '' IF-MIB::linkDown ifIndex i 2{{< /highlight >}}
+
+Here’s a quick explanation for those of you who are curious about what we’re doing. From the `snmptrap` man pages, this is the syntax of the command:
+
+{{< highlight shell >}}
+ snmptrap -v [2c|3] [COMMON OPTIONS] [-Ci] AGENT uptime trap-oid [OID TYPE VALUE]...{{< /highlight >}}
+
+So our command does the following:
+Sets the version (`-v`) to 2c
+Sets the community string (`-c`) to “sensutest”
+Specifies the host to send the trap to (`localhost:1062`)
+Sets the uptime ( ‘’ in our case, as it’s not necessary that we have the uptime for the test)
+Provides a shortened MIB (`IF-MIB::linkDown`)
+Provides the `ifIndex` type with an integer value of `2` (i.e., the 2nd port on a device)
+
+That’s it. That’s all we need to generate a trap. Running the command will give you no confirmation via the command line, but you should see something similar to the following in your Sensu Enterprise Dashboard:
+
+![snmp link down](/images/snmp_link_down.PNG)
+
+And in detail:
+
+![snmp link down detail](/images/snmp_link_down_detail.PNG)
+
+To resolve the alert, we pass a similar trap command:
+
+{{< highlight shell >}}
+sudo snmptrap -v 2c -c sensutest localhost:1062 '' IF-MIB::linkUp ifIndex i 2{{< /highlight >}}
+
+Which yields:
+
+![snmp link up](/images/snmp_link_up.PNG)
+
+You can then delete the result from the dashboard. 
+
+# Additional SNMP Extension Options (#additional-snmp-extension-options)
```

Comment:
I think the anchor should be wrapped in `{}` instead of `()`, it's rendering on the page for me:

![2018-03-22 at 11 23 am](https://user-images.githubusercontent.com/148017/37786944-83fb09d8-2dc3-11e8-9c41-ee0800033d5a.png)

Reply:
✔️